web/src/app/util/timeFormat.js

```diff
@@ -1,5 +1,15 @@
 import { Interval, DateTime } from 'luxon'
 
+// fortmateTimeLocale returns a formatted time stamp from a given ISO string
+// @params: iso: string, type: 'short' | 'full'
+// in a full, medium, or short locale. Defaults to medium.
+export function formatTimeLocale(iso, type) {
+  let locale = DateTime.DATETIME_MED
+  if (type === 'short') locale = DateTime.DATETIME_SHORT
+  if (type === 'full') locale = DateTime.DATETIME_FULL
+  return DateTime.fromISO(iso).toLocaleString(locale)
+}
+
```

Comment:
I think breaking this out is unnecessary indirection, it's just mapping arbitrary strings to the already defined constants in `DateTime` but doing no work itself. So instead of knowing to use `DATETIME_FULL` the caller has to read this file and know to pass `full` rather than just reading the luxon documentation.

Just call `DateTime.fromISO(iso).toLocaleString(DateTime.DATETIME_FULL)` from the site -- a little copying is better than a little dependency. If we were doing alterations or something it would warrant it's own function, but since this is a pass-through it doesn't give us much.

Reply:
Gotcha, sounds good — My main reasoning was for maintenance. If down the road we change the timestamps in one spot we would likely want to do the same in other places, based on our discussions of consistent timestamp display

Reply:
The issue is we're just swapping the constants `DateTime.DATETIME_SHORT` with `'short'` the format is already dictated by the locale, not by us, which is why I don't think the indirection is useful.

If we were writing our own format strings, I'd agree, but since we're using locale I think it's better to rely on luxon documentation than both ours and luxon.

As for maintenance, readability is going to help even if writing it takes slightly longer the first time. The luxon call tells you the expected format and you can look directly at their documentation without opening another file to find out we're using FULL and SHORT
`DateTime.fromISO(iso).toLocaleString(full ? DateTime.DATETIME_FULL : DateTime.DATETIME_SHORT)`

Reply:
You make a good point: it's a lot easier to use the luxon API directly than searching our utils for something that may or may not be there. We'll just have to stay on top of ourselves if/when we update timestamp formats in the future.